csrc/id_model/id_graphs.cpp

```diff
@@ -971,7 +972,8 @@ void IterDomainGraphs::build(
   idGraph(IdMappingMode::ALMOSTEXACT).removeTrivialExprs();
 
   // Only build loop map during lowering
-  if (FusionGuard::getCurFusion()->isA<kir::Kernel>()) {
+  // TODO: make this configurable
+  if (true || FusionGuard::getCurFusion()->isA<kir::Kernel>()) {
```

Comment:
Could you just pass the fusion into a kernel and pass it in instead of bypassing this check?

Reply:
It's possible but it's more convenient for testing if we could directly build ID graphs with the fusion as we can directly reference domains and tensors. I think we would need to make this part more flexible. Even in the kernel IR sometimes we may just want to build exact maps, etc.